src/filtered-list.ts

```diff
@@ -0,0 +1,65 @@
+import { List } from '@material/mwc-list';
+import { TextField } from '@material/mwc-textfield';
+import {
+  css,
+  customElement,
+  html,
+  property,
+  query,
+  TemplateResult,
+} from 'lit-element';
+
+@customElement('filtered-list')
+export class Filterlist extends List {
+  @property()
+  searchFieldLabel!: string;
+
+  @query('mwc-textfield') searchField!: TextField;
+
+  onFilterInput(): void {
+    this.items.forEach(item => {
+      const value: string =
+        item.innerText +
+        '\n' +
+        Array.from(item.children)
+          .map(child => child.innerHTML)
+          .join('\n');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/openscd/open-scd/pullRequest?prid=7203429)
```suggestion
          .join("\n");
```